scripts/chandra/README.md

```diff
@@ -0,0 +1,44 @@
+# Download Chandra X-ray spectral Data
+
+This dataset consists of reduced X-ray spectra for sources in the [Chandra Source Catalog](https://cxc.cfa.harvard.edu/csc/), 
+which is the repository of all sources detected by the Chandra X-ray Observatory. The catalog contains many data files,
+and here we will be adding only the spectra. Spectral files consist of three types of files: the Pulse Height Amplitude (PHA)
+file, which records the charge per pixel in the CCD, and two spectral response files the ARF, and the RMF, that together
+account for the telescopes effective area as a function of energy, quantum efficiency, etc. The scripts here will download the
+three types of files, but then will processs the PHA files using the response files, and only produce the final spectra. 
+
+## Downloading the data
+
+The script 'download_script.py' downloads the spectral files directly from the Chandra server, using the [CLI protocol](https://cxc.cfa.harvard.edu/csc/cli/).
+Here is how to use it:
+
+```
+> python download_script.py [Minumum number of source counts] [Minimum signal to noise] [Maximum off-axis angle] [Name of file] [Output directory]
+```
+
+The first three arguments are the number of X-ray counts in the source detection, the signal to noise of the detection,
+and the angle from the center of the field (the PSF degrades in a X-ray telescope far from the center of the field).
+The lower the number of counts or the signal to noise, the more source spectra are dowloaded, because dimmer sources
+are also selected. Recommended values for a short test are: 4000,80,1. To download the majority of the sources: 40,5,10.
+"Name of file" refers to the catalog file that is created.
+
+## Installing dependencies
+
+In order to process the spectrum using the response files, the [Sherpa](https://cxc.cfa.harvard.edu/sherpa/) software is required. To install it:
```

Comment:
This should already be installed from the `dset-requirements.txt`. It should either be updated to state "to check if ... is installed ..." or removed.